### ISSUE TITLE
WCAG 1.4.3: Elements must meet minimum color contrast ratio thresholds (.list-number.font-family-granville)

### ISSUE THREAD

**Issue by ryanbwork** [ryanbwork/assessment-filing#1]
#### Issue
`Elements must meet minimum color contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Ashley Black Guru | FasciaBlaster® - Self-Use Fascia Tool](https://www.ashleyblackguru.com/?srsltid=AfmBOooQ9DFiIM-90qk1UeY6F5PE3b6IOZZtgXqrqp8uNQ178mt2lN8d)
#### Element path
.list-item.justify-content-around.d-flex:nth-child(1) > .list-number.font-family-granville
#### Snippet
`<span class="list-number d-block font-family-granville">01</span>`
#### Environment
Chrome version 128.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.44.0 (axe-core 4.9.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
